src/utils.jl

```diff
@@ -0,0 +1,14 @@
+# general utility functions
```

Comment:
hmm... do we want to encourage this pattern?

Reply:
... lets get back to this after the Graph-Partition refactor

Reply:
Maybe we should throw away utils.jl, for now atleast. Currently after the refactor Graph and Partition both still take in the filepath, which I still think is a bit messy, but I am willing to live with it........ for now! Will keep thinking about this